### PR TITLE
* Use path.resolve for frameworkPath

### PR DESCRIPTION
**Pull request by Fatme**
* Don't delete frameworkPath directory

### CODE REVIEW

File: lib/services/platform-service.ts [hunk 1]
```diff
@@ -92,12 +93,6 @@ export class PlatformService implements IPlatformService {
 			platformData.platformProjectService.createProject(platformData.projectRoot, frameworkDir).wait();
 			var installedVersion = this.$fs.readJson(path.join(frameworkDir, "../", "package.json")).wait().version;
 
-			if(options.frameworkPath && !options.symlink) {
```
Comment (rosen-vladimirov) [inline lib/services/platform-service.ts]: why is this code deleted?

**Comment by ErjanGavalji** [comment]
test this please

**Comment by ErjanGavalji** [comment]
test this please

**Comment by rosen-vladimirov** [comment]
:+1:  after green build

**Comment by ErjanGavalji** [comment]
:+1:

**Pull request merged**